assets/js/util/escape-uri.ts

```diff
@@ -26,11 +26,18 @@
  *
  * @since 1.11.0
  *
- * @param {string[]} strings The array of static strings in the template.
- * @param {...*}     values  The array of expressions used in the template.
  * @return {string} Escaped URI string.
  */
-export function escapeURI( strings, ...values ) {
+export function escapeURI(
+	/**
+	 * The array of static strings in the template.
```

Comment:
Strange that this flags the inner comment as if it's a doc block for the function